tests/integration/kubernetes/confidential/k8s-confidential.bats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] ) && skip "Test not supported yet."
+
+	pod_name="confidential-unencrypted-test"
+	container_name="confidential-unencrypted-container"
+	get_pod_config_dir
+
+	k8s_generate_service_yaml "${TEST_DIR}/${pod_name}.yaml" "${IMAGE_REPO}:unencrypted"
+	k8s_yaml_set_annotation "${TEST_DIR}/${pod_name}.yaml" "io.katacontainers.config.pre_attestation.enabled" "false"
+}
+
+@test "Test unencrypted confidential container launch success." {
+  # Start the service/deployment/pod
+  esudo kubectl apply -f "${TEST_DIR}/sev-unencrypted.yaml"
+  
+  # Retrieve pod name, wait for it to come up, retrieve pod ip
+  pod_name=$(esudo kubectl get pod -o wide | grep ${pod_name} | awk '{print $1;}')
+
+  # Check pod creation
+  kubectl wait --for=condition=Ready --timeout=20 pod "$pod_name"
+
+  pod_ip=$(esudo kubectl get pod -o wide | grep ${pod_name} | awk '{print $6;}')
+
+  print_service_info
```

Comment:
Where's this function defined?
Same comment as the ones above.

Reply:
Removed.